samples/web/asp-web-application/Core_8/WebApp/Pages/Index.cshtml.cs

```diff
@@ -4,46 +4,38 @@
 using Microsoft.AspNetCore.Mvc.RazorPages;
 using NServiceBus;
 
-namespace WebApp.Pages
+namespace WebApp.Pages;
+
+[IgnoreAntiforgeryToken]
+public class IndexModel(IMessageSession messageSession) : PageModel
 {
-    [IgnoreAntiforgeryToken]
-    public class IndexModel : PageModel
+    public string ResponseText { get; set; }
+
+    public async Task<IActionResult> OnPostAsync(string textField)
     {
-        IMessageSession messageSession;
+        if (string.IsNullOrWhiteSpace(textField))
+        {
+            return Page();
+        }
 
-        public string ResponseText { get; set; }
+        #region ActionHandling
 
-        public IndexModel(IMessageSession messageSession)
+        if (!int.TryParse(textField, out var number))
         {
-            this.messageSession = messageSession;
+            return Page();
         }
-
-        public async Task<IActionResult> OnPostAsync(string textField)
+        var command = new Command
         {
-            if (string.IsNullOrWhiteSpace(textField))
-            {
-                return Page();
-            }
-
-            #region ActionHandling
-
-            if (!int.TryParse(textField, out var number))
-            {
-                return Page();
-            }
-            var command = new Command
-            {
-                Id = number
-            };
+            Id = number
+        };
 
-            var sendOptions = new SendOptions();
-            sendOptions.SetDestination("Samples.AsyncPages.Server");
+        var sendOptions = new SendOptions();
+        sendOptions.SetDestination("Samples.AsyncPages.Server");
```

Comment:
Any idea why this sample was doing routing this way and not at the endpoint configuration level?

Reply:
Not sure but there is no config file so perhaps to avoid adding one?